### ISSUE TITLE
About the results on Fine-grained datasets: aircraft.

### ISSUE THREAD

**Issue by rain305f** [nayeemrizve/TRSSL#7]
1. I reprocted your code on aircraft as your setting. But my reprocted results is Seen: 60.98% Unseen: 36.83%,All:48.90; But in your paper, your result is 
![YBO_ 51{HJUW H8( ZQ)L_2](https://user-images.githubusercontent.com/128030543/229675056-e0d868cf-8a54-4df5-bf92-ff2b7ac57181.png)

2. About the learning rate adjust strategy: "We use a cosine annealing based learning rate scheduler accompanied by a linear warmup, where we set the base learning rate to 0.1 and set the warmup length to 10 epochs". In the latest version of code, you remove it. It does affect performance. So should this operation apply all dataset?

3. About the fine-grained datasets preprocess:
    In your papers, the preprocess is as follows:
> The input resolution of CIFAR-10 and CIFAR-100 images is 32×32; Tiny ImageNet images are slightly larger, i.e., 64×64. For the fine-grained datasets the images vary in size and aspect ratio. Therefore, for computational efficiency, we pre-process the images for fine-grained datasets and resize them to 256×256 resolution; this pre-processing operation is performed for both train and test images in all of our experiments.

 But in your code, the data transformer is as follows:

>      self.transform_train = transforms.Compose([
>             transforms.RandomResizedCrop(224, (0.5, 1.0)),
>             transforms.RandomHorizontalFlip(),
>             transforms.RandomApply([transforms.ColorJitter(0.4, 0.4, 0.2, 0.1)], p=0.5),
>             transforms.RandomGrayscale(p=0.2),
>             transforms.RandomApply([GaussianBlur([0.1, 2.0])], p=0.2),
>             transforms.ToTensor(),
>             transforms.Normalize(imgnet_mean, imgnet_std),
>         ])

`
`
       

```
 self.transform_val = transforms.Compose([
            transforms.Resize(256),
            transforms.CenterCrop(224),
            transforms.ToTensor(),
            transforms.Normalize(mean=imgnet_mean, std=imgnet_std)
        ])
```

So the size of inputs is 224\*224 or 256\*256? Thanks~
